repairnator/scripts/launch_dockerpool.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+# Use to create args in the command line for optional arguments
+function ca {
+  if [ -z "$2" ];
+  then
+      echo ""
+  else
+    if [ "$2" == "null" ];
+    then
+        echo ""
+    else
+        echo "$1 $2 "
+    fi
+  fi
+}
+
+if [ "$#" -eq 0 ]; then
+    echo "A list of build ids must be provided"
+    exit -1
+fi
+
+if [ ! -f $1 ]; then
+    echo "The list of build ids must be an existing file ($1 not found)"
+    exit -1
+fi
+
+REPAIRNATOR_BUILD_LIST=$1
+NB_LINES=`wc -l < $REPAIRNATOR_BUILD_LIST`
+
+if [[ $NB_LINES == 0 ]]; then
+    echo "No build has been found. Stop the script now."
+    exit 0
+else
+    echo "$NB_LINES builds have been found."
+fi
+
+if [ "$#" -eq 1 ]; then
```

Comment:
What's the purpose of this if? If `$# > 1` you don't execute those lines? There's a problem here.

Reply:
> If $# > 1 you don't execute those lines?

Yes. It's because the `launch_dockerpool.sh` (and also `launch_scanner.sh`) can be executed from `launch_repairnator.sh`. So, if this is the case, the first stuff to do, like to init and create the structure, should be done only once (i.e. in `launch_repairnator.sh`), thus those lines can't be executed. See in `launch_repairnator.sh` how I call `launch_dockerpool.sh` and `launch_scanner.sh`. There is an extra arg to control what I just described. I found no way better to do that. If you have any suggestion, please, let me know.

Reply:
In that case, 
  1. We should be calling only init_scripts.sh which should contain also content of create_structure.sh
  2. When init_script.sh is called, it should initialize a variable like "REPAIRNATOR_INITIALIZED" and you should test the value of this variable.

Reply:
Perfect.